### ISSUE TITLE
Feature or bug? Local component child of imported parent not printed?

### ISSUE THREAD

**Issue by kerimoyle** [cellml/libcellml#609]
Parsing a model and then printing it gives different results:
**Parsing this:**
```
<?xml version="1.0" encoding="UTF-8"?>
<model xmlns="http://www.cellml.org/cellml/2.0#" name="import_example">
  <import xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="importedModel.cellml">
    <component component_ref="importMe" name="importedComponent"/>
  </import>
  <component name="localComponent">
    <variable name="x" units="dimensionless" interface="public_and_private"/>
  </component>
  <encapsulation>
    <component_ref component="importedComponent">
      <component_ref component="localComponent"/>
    </component_ref>
  </encapsulation>
</model>
```
... gives this model (NB: components are included in their encapsulation structure)
```
 MODEL: 'import_example'
    UNITS: 0 custom units
    COMPONENTS: 1 components
        [0]: importedComponent
            VARIABLES: 0 variables
            COMPONENT importedComponent has 1 child components:
                [0]: localComponent
                    VARIABLES: 1 variables
                        [0]: x [dimensionless]
```

... which is serialised to this:
```
<?xml version="1.0" encoding="UTF-8"?>
<model xmlns="http://www.cellml.org/cellml/2.0#" name="import_example">
  <import xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="importedModel.cellml">
    <component component_ref="importMe" name="importedComponent"/>
  </import>
  <encapsulation>
    <component_ref component="importedComponent">
      <component_ref component="localComponent"/>
    </component_ref>
  </encapsulation>
</model>
```
... ie: the `localComponent` definition is missing from the printed version.  The parsed model is valid, and neither the parser nor the printer have any issues.  You can see this behaviour in #591 ... but it's now to do with the printer not the parser ...

**Comment by nickerso** [comment]
definitely a bug. But this is fixed in #591, right?

**Comment by kerimoyle** [comment]
Yes.
